pkgs/servers/misc/podgrab/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "podgrab";
+  version = "2021-04-14"; # No official version tags
+
+  src = fetchFromGitHub {
+    owner = "akhilrex";
+    repo = pname;
+    rev = "3179a875b8b638fb86d0e829d12a9761c1cd7f90";
+    sha256 = "sha256-vhxIm20ZUi+RusrAsSY54tv/D570/oMO5qLz9dNqgqo=";
+  };
+
+  vendorSha256 = "sha256-xY9xNuJhkWPgtqA/FBVIp7GuWOv+3nrz6l3vaZVLlIE=";
+
+  postInstall = ''
+    mkdir -p $out/share/
+    ln -s -r "$src/client" "$out/share/"
+    ln -s -r "$src/webassets" "$out/share/"
+  '';
+
+  meta = with lib; {
+    description = "A self-hosted podcast manager to download episodes as soon as they become live";
+    homepage = "https://github.com/akhilrex/podgrab";
+    license = licenses.gpl3;
```

Comment:
Prefer the gpl3Plus or gpl3Only attributes; I think this one is gpl3Only because it's not called out as being "GPL 3 or any later version".